docs/tbdev_getting_started.ipynb

```diff
@@ -1,235 +1,272 @@
 {
```

Comment:
<p>This notebook is primarily run in Colab (users reach it from "Example Colab" on https://tensorboard.dev). I don't think we should significantly increase the length of the tutorial with this section.</p><p> </p><p>What if we changed this to a one line statement in the next section: "When running in a Jupyter notebook, use <code>tensorboard dev list</code> to authenticate".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>